test/fixtures/stdout-producer.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const add = 'test\n';
+
+var str = '';
+for (var i = 0; i < 100000; i++) {
```

Comment:
Suggestion: use `1e5` here.  Personally, I find scientific notation easier to read for large numbers.